### ISSUE TITLE
The 'substringof' OData function is no longer valid on v4.0

### ISSUE THREAD

**Issue by lesair** [devnixs/ODataAngularResources#122]
The 'substringof' OData function mentioned in the Readme is no longer valid for OData v4, as according to the URI Conventions it was renamed as 'contains'.

See section 5.1.1.5.2 of [the document](http://docs.oasis-open.org/odata/odata/v4.0/odata-v4.0-part2-url-conventions.html).

**Comment by ahmedeasa** [comment]
is there any workaround for this issue

**Comment by lesair** [comment]
Yes. Just use the new OData v4 'contains' function. It works well.

**Comment by ahmedeasa** [comment]
yes you are right it is working well for me, thank you
